templates/download/iot/index.html

```diff
@@ -132,10 +146,20 @@ <h3>Supported platforms</h3>
           <img src="https://assets.ubuntu.com/v1/db577ec7-tank-logo.svg" width="163" height="45" alt="Intel IEI TANK 870">
           <p><a href="/download/intel-iei-tank-870">Install Ubuntu Core 18&nbsp;&rsaquo;</a></p>
         </div>
+      </div>
+      <div class="row">
         <div class="col-3">
           <img src="https://assets.ubuntu.com/v1/2a6d29c9-snapdragon.svg" width="139" height="45" alt="Qualcomm Snapdragon">
           <p><a href="/download/qualcomm-dragonboard-410c">Install Ubuntu Core 18&nbsp;&rsaquo;</a></p>
         </div>
+        <div class="col-3">
+          <img src="https://assets.ubuntu.com/v1/8a8c250f-xilinx-logo.svg" width="163" height="45" alt="Xilinx">
+          <p><a href="/download/xilinx">Install Ubuntu Core 20&nbsp;&rsaquo;</a></p>
+        </div>
+        <div class="col-3">
+          <p>Can't find your board of choice?</p>
+          <a class="p-button--positive" href="">Get in touch</a>
```

Comment:
This should have the model revoke class?

Reply:
I forgot Thibaut replied to me about adding a general contact us for now - there was no contact us or thank you for download so I've added one but used the form id from [here](4126) for the supported platforms get in touch as Thibaut mentioned he had the ID [here](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit?disco=AAAAIjbYdgM)